kamailio/db_scripts/db_extra_check.sql

```diff
@@ -0,0 +1,23 @@
+
+do_db_extra_check() {
+
+# location
+KazooDB -db ${DB_CURRENT_DB} "delete from location where socket not like 'udp:%';"
+KazooDB -db ${DB_CURRENT_DB} "delete from location  where expires > 0 and datetime(expires, 'unixepoch') < datetime('now', '-10 seconds');"
```

Comment:
Could you point me to issue that is fixed by force delete regitstration and related info

Reply:
This also executed before kamaillio started. Now understand